src/digraph.rs

```diff
@@ -174,6 +174,10 @@ use super::dag_algo::is_directed_acyclic_graph;
 /// :param attrs: An optional attributes payload to assign to the
 ///     :attr:`~.PyDiGraph.attrs` attribute. This can be any Python object. If
 ///     it is not specified :attr:`~.PyDiGraph.attrs` will be set to ``None``.
+/// :param int initial_node_count: The graph will be allocated with enough capacity to store this
+///     many nodes before needing to grow (default 0: no overallocation).
+/// :param int initial_edge_count: The graph will be allocated with enough capacity to store this
+///     many edges before needing to grow (default 0: no overallocation).
```

Comment:
I would add here explicitly that this does not prepopulate any nodes or edges. It's all about runtime performance for when you know the size of the graph ahead of time. I think we should be a bit more explicit here for python users who aren't necessarily familiar with using this kind of pattern.

Reply:
Done in b9f564a4.